doc/mainpage.dox

```diff
@@ -223,5 +223,106 @@ before the measurement framework or in a particular Plugin/Algorithm not knowing
 failure modes.  This means that Plugin/Algorithm implementations should generally have a global try/catch
 block that re-throwns lower-level exceptions corresponding to known failure modes as @ref MeasurementErrors.
 
+@section measBaseFlagHandlerInPython Using a FlagHandler with Python Plugins
+
+As mentioned above, a plugin measure() method is used to implement the code which performs
+measurements on individual detected sources.  It also records the results in the output
+source catalog. If an error occurs during measure(), the plugin is obliged to record the error by setting a
+flag in the source catalog.  As described above, this is often done by raising a MeasurementError and relying
+on the fail() method of the plugin to set the error flags.
+
+<B>
+How to Define a FlagHandler in Python:
+</B>
+
+If you have looked at any of the C++ implementations of plugins in meas_base, you will notice that the C++
+algorithms implement their fail() method using a C++ class called lsst::meas::base::FlagHander.  In this
+section, we will show you how to do the same thing in Python using the addFlagHandler decorator.
+
+First examine the code testFlagHandler.py in the meas_base tests directory. This unit test defines a
+plugin written in Python which illustrates several useful techniques about using the FlagHander.
+Note first the line decorating the class definition, which reads like this:
+
+@code{.py}
```

Comment:
Instead of only showing the class decorator line, I think it would be useful to show an entire framework of a measurement plugin. Use `# ...` comments to indicate where details are omitted for brevity. Showing a whole class example puts the entire measurement plugin in context, and gives you something to refer to in subsequent paragraphs.

```
@addFlagHandler(("flag", "General Failure error"),
                ("flag_containsNan","Measurement area contains a nan"),
                ("flag_edge", "Measurement area over edge"))
class PythonPlugin(SingleFramePlugin):

    ConfigClass = PythonPluginConfig

    def __init__(self, config, name, schema, metadata):
        SingleFramePlugin.__init__(self, config, name, schema, metadata)
        # self.flagHander and self.errEnum are created by the addFlagHandler
        # class decorator

        # ...

    def measure(self, measRecord, exposure):
        # ...

    def fail(self, measRecord, error=None):
        if error is None:
            self.flagHandler.handleFailure(measRecord)
        else:
            self.flagHandler.handleFailure(measRecord, error.cpp)
```